### PR TITLE
#175636830 setup sequelize 

### PR DESCRIPTION
**Pull request by tytyne**
#### What does this PR do?
- The PR setup sequelize in the project
#### Description of Task to be completed?
- creating sequelizerc file 
- creating migrations folder
- creating seeders folder
- creating models folder 

#### How should this be manually tested?
- ` git clone https://github.com/atlp-rwanda/tech-strikers-bn-backend  `
- git checkout ch-setup-sequelize-175636830
- npm run start 
#### Any background context you want to provide?
no
#### What are the relevant pivotal tracker stories?
[#175636830](https://www.pivotaltracker.com/story/show/175636830)
#### Screenshots (if appropriate)
sequelizerc file
sequelize

### CODE REVIEW

**Comment by Duncanwar** [comment]
In How it is manually tested you can ass the sequelize command

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 # Edit at https://www.gitignore.io/?templates=node
 
 ### Node ###
+package-lock.json
```
Comment (kipropbrian) [inline .gitignore]: Remove package.lock from gitignore

File: package.json [hunk 2]
```diff
@@ -24,14 +25,23 @@
...
+    "pg": "^8.5.0",
     "request": "^2.87.0",
+    "sequelize": "^6.3.5",
     "underscore": "^1.9.1"
   },
   "devDependencies": {
+    "@babel/cli": "^7.12.1",
+    "@babel/core": "^7.12.3",
+    "@babel/node": "^7.12.6",
+    "@babel/preset-env": "^7.12.1",
```
Comment (kipropbrian) [inline package.json]: These are not dev dependencies.

**Review by kipropbrian — CHANGES_REQUESTED** [review]
Please check the comments and rebase with develop again

File: package.json [hunk 3]
```diff
@@ -1,37 +1,38 @@
 {
-  "name": "express-authorshaven",
+  "name": "tech-strikers-bn-backend",
   "version": "1.0.0",
-  "description": "A Social platform for the creative at heart",
+  "description": "[![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)",
```
Comment (geraudirad) [inline package.json]: I think this should be the project description.
Hound badge has to be added in the readme.

**Review by geraudirad — CHANGES_REQUESTED** [review]

**Pull request closed without merging**